neuvue_project/workspace/analytics.py

```diff
@@ -111,8 +111,10 @@ def create_stats_table(pending_tasks, closed_tasks):
         for namespace, namespace_df in df.groupby("namespace"):
             n_tasks = len(namespace_df)
             m = namespace_df[status].min()
-            average_event_time = (m + (namespace_df[status] - m)).mean().to_pydatetime()
-            changelog.append((average_event_time, n_tasks, status, namespace))
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+                average_event_time = (m + (namespace_df[status] - m)).mean().to_pydatetime()
+                changelog.append((average_event_time, n_tasks, status, namespace))
```

Comment:
Why are we ignoring warnings here?

Reply:
a previous team member hannah gooden reported annoying warnings here
https://github.com/aplbrain/neuvue-app/issues/382